src/test/java/com/exasol/adapter/dialects/rls/AbstractRowLevelSecurityIT.java

```diff
@@ -204,6 +205,31 @@ void testGroupRestrictedTableWithUnauthorizedUser() throws SQLException {
         assertThat(exception.getMessage(), containsString("E-VS-RLS-JAVA-7"));
     }
 
+    @Test
+    void testGroupAndRoleRestrictedTable() {
```

Comment:
Two test cases in one method.

Reply:
I wanted to avoid copying the test, it's quite long

Reply:
Split them into 2 tests